### DISCUSSION TITLE
Reset de l'App

### DISCUSSION THREAD

**Discussion by YvesBr** [seb-cogez/QCMCam#4] [Ideas]
Bonjour,
Je viens de tester QcmCam (formidable) avec mon smartphone. 
J'ai essayé de l'utiliser en mode "hyper light" : une seule question appelée "Question" et 4 réponses appelée A,B,C,D. L'idée est de poser la question à l'oral (ou de la projeter) et de faire voter pour l'une des 4 réponses. La solution est donnée à l'oral et le debrief aussi. Du coup seul le mode "voir les réponses" est utilisé.
Malheureusement pour poser une autre question (oralement) je dois "remettre à zéro" l'app sur mon smartphone  (recharger la page et re-scanner le code). Je ne pense pas (ou je n'ai pas trouvé comment faire) que l'app permette de reposer la même question.
Peut être qu'une version simplifiée de l'app permettrait d'envisager cet usage ? (Un simple décompte des marqueurs)

**Comment by seb-cogez** [comment]
Bonjour,
Ce n'est effectivement pas programmé tel quel via le smartphone. Je vais réfléchir à cela pour la prochaine mouture, mais je ne sais pas encore quand j'aurai du temps à y consacrer.
Merci pour cette suggestion.
